shared/components/graphs/graphs.component.js

```diff
@@ -215,8 +231,31 @@ class GraphsComponent extends Panel {
     }
   }
 
+  get category_average_byline(){
+    let graphs = this;
+    switch (graphs.current_route_name){
+      case 'Travel':
+        return graphs.t('summaries.average_travel_footprint')
+      case 'Home':
+        return graphs.t('summaries.average_home_footprint')
+      case 'Food':
+        return graphs.t('summaries.average_food_footprint')
+      case 'Shopping':
+        return graphs.t('summaries.average_shopping_footprint')
+      case 'TakeAction':
+        return graphs.t('summaries.average_action_savings')
+      default:
+        return graphs.t('summaries.average_footprint')
+    }
+  }
+
   get display_total_footprint(){
     let graphs = this;
+    if (graphs.current_route_name === 'GetStarted') {
+      graphs.state.show_total = false;
+    } else {
+      graphs.state.show_total = true;
+    }
```

Comment:
This route condition was actually redundant. Anyways made sure that state is not changed in getters:
`  componentDidMount() {
    let graphs = this;
    if (window.outerWidth < 992) {
      graphs.setState({
        show_chart: false
      });
    }`